packages/dd-trace/src/config.js

```diff
@@ -79,6 +79,11 @@ class Config {
       process.env.DD_RUNTIME_METRICS_ENABLED,
       false
     )
+    const DD_TRACE_SQL_COMMENT_INJECTION_MODE = coalesce(
+      options.sqlInjectionMode,
```

Comment:
"sql injection" has a lot of connotations that we'd best avoid here. Since this is specifically for DBM, maybe we can call this `dmInjectionMode`?

Reply:
This is a cross-language feature, is it not? Have we not had a discussion about what the config names should be across languages? If not, we probably _should_ before landing this.

Reply:
this has changed to `dbmPropagationMode` and `DD_DBM_PROPAGATION_MODE` according to the [doc](https://docs.google.com/document/d/1hlp3CuIRE-os2zYgc0wYOS8kPtCKqlrGL7aepbHHDaU/edit#).